docs/development.md

```diff
@@ -67,50 +68,60 @@ bundle exec foreman start
 The files that webpack handles are located in `webpack/` directory found in Foreman, Katello,
 and plugin root directories. If you are editing any files in `webpack/` and want to have your changes refresh automatically, you will need a webpack server running.
 
 Because we are using a webpack server in conjunction with a rails server, there are different ways of starting a server depending on your needs and preferences. The following are instructions for starting the server using a base `centos9-katello-devel` box as a starting point.
 
 #### Run a rails and webpack server together using `foreman start`
-- Run `bundle exec foreman start` in `~/foreman`
-- Navigate to `https://centos9-katello-devel.<hostname>.example.com/` where `<hostname>` is the shortname of your hypervisor (machine your VM is running on) and accept the self-signed certs.
-- Accept the self-signed certs on port 3808 at `https://centos9-katello-devel.<hostname>.example.com:3808`.
-- Everything should be set for you to run `bundle exec foreman start` to start your dev server as needed.
+
+* Run `bundle exec foreman start` in `~/foreman`
+* Navigate to `https://centos9-katello-devel.<hostname>.example.com/` where `<hostname>` is the shortname of your hypervisor (machine your VM is running on) and accept the self-signed certs.
+* The initial page load after startup might take a while.
+* Everything should be set for you to run `bundle exec foreman start` to start your dev server as needed.
 
 NOTE: The `foreman` in `foreman start` is actually [this gem](https://github.com/ddollar/foreman) and not our `foreman`. It
 doesn't allow STDIN, which means it doesn't support local debugging. To use a debugger with `foreman start`, you can use
 remote debugging with a tool like [pry-remote](https://github.com/Mon-Ouie/pry-remote). There is a helpful blog post about
-remote debugging [here](http://blog.honeybadger.io/remote-debugging-with-byebug-rails-and-pow/) with more information.
+remote debugging [here](https://blog.honeybadger.io/remote-debugging-with-byebug-rails-and-pow) with more information.
 
 #### Run the rails server without a webpack server running
+
 If you don't need to do any development in the `webpack/` directories, you can turn the webpack server off and run only the rails server if desired.
 
-- Run `foreman-installer --katello-devel-webpack-dev-server false` to disable the webpack server. You can also set this installer option
+* Run `foreman-installer --katello-devel-webpack-dev-server false` to disable the webpack server. You can also set this installer option
 in your boxes.yaml entry to configure this on box creation.
-- Start the rails server with `bundle exec foreman start rails` in `~/foreman`
-  - Alternatively, you can start the rails server with `bundle exec rails s`
+* Start the rails server with `bundle exec foreman start rails` in `~/foreman`
+    * Alternatively, you can start the rails server with `bundle exec rails s`
```

Comment:
We don't necessarily need to document this, but  this is my one-liner I use to avoid the foreman gem altogether:
```
source .env && BIND=0.0.0.0 bundle exec puma -w 3 -p $PORT --preload
```

Reply:
does that have advantages over `bundle exec rails s`?

Reply:
It's nice to be able to set the number of workers, that's mostly it.

Reply:
My thought is, why not both then, as `bundle exec rails s` is so much shorter, but can also replace it.